elasticsearch/src/main/java/org/apache/calcite/adapter/elasticsearch/ElasticsearchSchemaFactory.java

```diff
@@ -49,15 +58,37 @@ public Elasticsearch5SchemaFactory() {
       final Map<String, Integer> coordinates =
           mapper.readValue((String) map.get("coordinates"),
               new TypeReference<Map<String, Integer>>() { });
+
+      final RestClient client = connect(coordinates);
+
       final Map<String, String> userConfig =
           mapper.readValue((String) map.get("userConfig"),
               new TypeReference<Map<String, String>>() { });
+
       final String index = (String) map.get("index");
-      return new Elasticsearch5Schema(coordinates, userConfig, index);
```

Comment:
What's with the userConfig map? Before, a user was able to specify driver settings through this like e.g. `bulk.flush.max.actions`, `bulk.flush.max.size.mb` and `bulk.flush.interval.ms`. Are there any settings that we could allow users to do through this?

Reply:
Those changes were applicable only for legacy transport client. New low-level rest client is [configured programatically](https://www.elastic.co/guide/en/elasticsearch/client/java-rest/current/_common_configuration.html). We may add some parameters in future (like timeout).